auto/capability

```diff
@@ -0,0 +1,47 @@
+
+# Copyright (C) Igor Sysoev
+# Copyright (C) NGINX, Inc.
+
+NXT_HAVE_LINUX_CAPABILITY=NO
+NXT_HAVE_SOLARIS_PRIVILEGE=NO
+
+# Linux capability
+
+nxt_feature="Linux capability"
+nxt_feature_name=NXT_HAVE_LINUX_CAPABILITY
+nxt_feature_test="#include <linux/capability.h>
+                  #include <unistd.h>
+                  #include <sys/syscall.h>
+
+                  int main() {
+                      struct __user_cap_header_struct hdr;
+                      hdr.version = _LINUX_CAPABILITY_VERSION; 
+                      syscall(SYS_capget, &hdr, 0);
+                      return 0;
+                  }"
+. auto/feature
+
+if [ $nxt_found = yes ]; then
+    NXT_HAVE_LINUX_CAPABILITY=YES
+fi
+
+if [ $nxt_found = no ]; then
+    # Solaris Privileges
```

Comment:
Let's drop the Solaris support from this PR. We can add it later if there will be any requests, but right now it will be waste of time and resources.